bottle_app.py

```diff
@@ -61,6 +61,7 @@
 import bottle
 from bottle import request
 from validate_email_address import validate_email
+from tracker import *
```

Comment:
We never do a `from <anything> import *`.
Please see: https://stackoverflow.com/questions/2386714/why-is-import-bad#:~:text=Because%20it%20puts%20a%20lot,thing%20was%20imported%20(readability).